README.md

```diff
@@ -54,18 +54,17 @@ winget install Microsoft.PowerToys -s winget
 
 ### Other install methods
 
-There are [community driven install methods](./doc/unofficialInstallMethods.md) such as Chocolatey and Scoop. If these are your preferred install solutions, you can find the install instructions there.
+There are [community-driven install methods](./doc/unofficialInstallMethods.md) such as Chocolatey and Scoop. You can find the install instructions if these are your preferred install solutions.
 
 ## Contributing
 
-This project welcomes contributions of all types. Help spec'ing, design, documentation, finding bugs are ways everyone can help on top of coding features / bug fixes. We are excited to work with the power user community to build a set of tools for helping you get the most out of Windows.
+This project welcomes contributions of all types. Help spec'ing, design, documentation, and finding bugs are ways everyone can help on top of coding features/bug fixes. We are excited to work with the power user community to build a set of tools for helping you get the most out of Windows.
 
-We ask that **before you start work on a feature that you would like to contribute**, please read our [Contributor's Guide](CONTRIBUTING.md). We will be happy to work with you to figure out the best approach, provide guidance and mentorship throughout feature development, and help avoid any wasted or duplicate effort.
+We ask that **before you start working on a feature you would like to contribute**, please read our [Contributor's Guide](CONTRIBUTING.md). We will happily work with you to figure out the best approach, provide guidance and mentorship throughout feature development, and help avoid wasted or duplicate effort.
```

Comment:
this IMO is a style thing,  I prefer "be happy to" as it inferring collaboration.

Reply:
i think I resolved that now.